orion/config.go

```diff
@@ -51,6 +51,8 @@ type Config struct {
 	Env string
 	// DefaultJSONPB sets jsonpb as the encoder/decoder for application/json request/response bodies
 	DefaultJSONPB bool
+	// NoDefaultInterceptors disables all default interceptors on all handlers
+	NoDefaultInterceptors bool
```

Comment:
Can we rename this to `EnableDefaultInterceptors` with a default value of `true`? Similarly, can we change the config to be `orion.EnableDefaultInterceptors` to better convey the type of the config.

Reply:
I don't agree with this. I think the empty value (false for a boolean) should be the default, and the naming of the flag should accommodate the default value.

e.g. if by default we want interceptors, the flag should be named `DisableInterceptors`, on the flipside if we wanted no interceptors by default, then the flag should be named `EnableInterceptors`.

Reply:
You're correct @benjaminheng . My suggestion was about renaming the var - the final name of course is up to the logic you've mentioned.

So in this case, `DisableDefaultInterceptors` with a `false` value would be good.

Reply:
For the config name, I'm also in favor of `DisableDefaultInterceptors`

Reply:
@achichen what do you think if we also separate the config for unary and stream interceptors? https://github.com/carousell/Orion/blob/master/orion/handlers/utils.go#L135 as they also have different interface functions to fill in back. In general, they may have different needs.

Reply:
@benjaminheng @anaghsys I've renamed the config to `DisableDefaultInterceptors`